setup.py

```diff
@@ -36,37 +36,38 @@ def clear_folder(path):
 
 def find_rgf_lib():
     if system() in ('Windows', 'Microsoft'):
-        exe_file = os.path.join(CURRENT_DIR, 'include/rgf/bin/rgf.exe')
+        exe_file = os.path.join(CURRENT_DIR, 'include', 'rgf', 'bin', 'rgf.exe')
```

Comment:
Delimitering by "," is better than "/"?
Could you explain why?

Reply:
`os.path.join()` creates platform-dependent path, it's better than hardcoding path in Unix/Windows style.

Reply:
```
import os

CURRENT_DIR = os.path.abspath(os.path.curdir)
os.path.join(CURRENT_DIR, 'include/rgf/bin/rgf.exe') 

>>> 'C:\\Users\\nekit\\include/rgf/bin/rgf.exe'
```

Reply:
Thanks! OK, LGTM!